app/views/admin/index.scala.html

```diff
@@ -322,7 +322,12 @@ <h1>Comments</h1>
                                     <td><a href='@routes.AdminController.userProfile(comment.username)'>@comment.username</a>
                                     </td>
                                     <td class = 'timestamp'>@comment.timestamp</td>
-                                    <td>@comment.gsvPanoramaId</td>
+                                    <td>
+                                        <a class="showCommentLocation" href="#" 
```

Comment:
And I feel like the entire `<a>` tag should probably be on one line... Looking for
```
<a blah blah blah>
    @comment.gsvPanoramaId
</a>
```

Reply:
This was the line that you requested be shortened to under 120 characters. I can put the whole tag back on one line, but it would exceed that.

Reply:
hmm, okay well right now it is still over 120 characters, looks like the 2nd line is at 160 (you have to include the spaces). and I'm seeing that I didn't fully describe what I meant. In the HTML, I think that it is okay-ish to have lines over 120 characters; it's kind of inevitable. BUT if you have a really long line, then I think that by default you shouldn't be stacking multiple tags to make it even longer. So the issue I was really getting at was that you had
```
<td><a blah blah>@comment.gsvPanoramaId</a></td>
```

and I think it's better to split into multiple lines if it goes over 120 characters (including spaces). So you want to spread the `td` and `a` and their closing tags:
```
<td>
    <a blah blah>
        stuff in the a tag
    </a>
</td>
```

Even if one of those lines becomes too long, I think this makes the most sense visually.

Reply:
Ohhh yes for sure. Sorry for the misunderstanding, that makes total sense.